README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+[Swarmkit](https://github.com/docker/swarmkit): Cluster management and
+orchestration features for swarm mode in Docker Engine 1.12 or later. See the
+[feature list](https://docs.docker.com/engine/swarm/).
+
+[Docker Engine swarm mode](https://github.com/docker/docker): CLI interface for
+[swarm mode](https://docs.docker.com/engine/swarm/#what-s-next).
+
+[Docker Swarm standalone](https://github.com/docker/swarm): An API proxy system
```

Comment:
Agree. We should have wording that focuses on the current repo here, and emphasize that this is the repo for standalone.